cnf-certification-test/lifecycle/suite.go

```diff
@@ -113,8 +114,7 @@ func testContainersImagePolicy(env *provider.TestEnvironment) {
 		for _, cut := range env.Containers {
 			logrus.Debugln("check container ", cut.Namespace, " ", cut.Podname, " ", cut.Data.Name, " pull policy, should be ", v1.PullIfNotPresent)
 			if cut.Data.ImagePullPolicy != v1.PullIfNotPresent {
-				s := cut.Namespace + ":" + cut.Podname + ":" + cut.Data.Name
-				badcontainers = append(badcontainers, s)
+				badcontainers = append(badcontainers, strings.Join([]string{cut.Namespace, cut.Podname, cut.Data.Name}, ":"))
```

Comment:
Shouldn't we use cut.String() instead? same comment for the logrus lines for instance @ line 115
badcontainers = append(badcontainers, cut.String())

Reply:
Yes that works too.  I'll make this use cut.String() instead.